src/cljs/rems/administration/administration.cljs

```diff
@@ -27,7 +27,8 @@
    [navbar/nav-link "/administration/licenses" (text :t.administration/licenses)]
    [navbar/nav-link "/administration/reports" (text :t.administration/reports)]
    [navbar/nav-link "/administration/blacklist" (text :t.administration/blacklist)]
-   [navbar/nav-link "/administration/organizations" (text :t.administration/organizations)]])
+   [navbar/nav-link "/administration/organizations" (text :t.administration/organizations)]
+   [navbar/nav-link "/administration/categories" (text :t.administration/categories)]])
```

Comment:
Should we place this right next to the catalogue items? Or link to it from the catalogue items admin page and not have it in the menu?

Reply:
moved categories link to catalogue items and catalogue item page(s) as "sub page" of sorts